.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  # Maintain dependencies for npm
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  # Maintain dependencies for Golang
+  - package-ecosystem: "gomod"
+    directory: "/"
```

Comment:
Isn't our go code all in `/src/go/…`? (There's two projects, `nerdctl-stub` and `wsl-helper`; does dependabot do the correct thing for both?)

Reply:
from what I understand `dependabot` finds packaging constructs and submit updates against them